<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerLoginWidget</class>
 <widget class="QWidget" name="ManagerLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ManagerLoginWidget
{

border: 3px solid gray;
border-radius: 40px;
background: white;
}
QLineEdit {
        border-radius: 4px;
        height: 25px;
        border: 1px solid rgb(111, 156, 207);
        background: white;
}
QLineEdit:enabled {
        color: rgb(84, 84, 84);
}
QLineEdit:enabled:hover, QLineEdit:enabled:focus {
        color: rgb(51, 51, 51);
}
QLineEdit:!enabled {
        color: rgb(80, 80, 80);
}

QPushButton{
        border-radius: 4px;
        border: none;
        width: 75px;
        height: 50px;
}
QPushButton:enabled {
        background: rgb(120, 170, 220);
        color: white;
}
QPushButton:hover {
	background: rgb(0, 78, 161);
}
</string>
  </property>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>219</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>219</width>
     <height>65</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>201</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_confirm">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>201</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_account"/>
    </item>
    <item row="0" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_account">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>账号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_password"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_password">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
